Signal/src/ViewControllers/ConversationView/ConversationInputTextView.m

```diff
@@ -44,13 +52,89 @@ - (instancetype)init
 
         self.text = nil;
 
-        //        _placeHolder = nil;
-        //        _placeHolderTextColor = [UIColor lightGrayColor];
+        self.placeholderView = [UILabel new];
+        self.placeholderView.text = NSLocalizedString(@"new_message", @"");
+        self.placeholderView.textColor = [UIColor lightGrayColor];
+        self.placeholderView.textAlignment = NSTextAlignmentLeft;
```

Comment:
Shouldn't the alignment be default to be RTL friendly?

Reply:
You're right!